docs/tutorial/part-six/index.md

```diff
@@ -1,31 +1,26 @@
 ---
-title: Transformer plugins
+title: ટ્રાન્સફોર્મર plugins
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-> This tutorial is part of a series about Gatsby’s data layer. Make sure you’ve gone through [part 4](/tutorial/part-four/) and [part 5](/tutorial/part-five/) before continuing here.
+> આ ટ્યુટોરીયલ Gatsby ડેટા સ્તર વિશેની શ્રેણીનો ભાગ છે. અહીં ચાલુ રાખતા પહેલા ખાતરી કરો કે [ભાગ ૪](/tutorial/part-four/) અને [ભાગ ૫](/tutorial/part-five/) પૂર્ણ કર્યો છે .
```

Comment:
`અહીં ચાલુ રાખતા પહેલા ખાતરી કરો કે` This is quite confusing. Please use some other words.